templates/monasca-persister-java.service.j2

```diff
@@ -1,6 +1,7 @@
 [Unit]
 Description=Monasca Persister
-After=kafka.service influxdb.service
```

Comment:
The persister will just restart over and over if kafka and influxdb are not on the same machine. In some cases they are and in others they aren't but since systemd handles it properly in either case it is nice to have these there to avoid some cases of many startup errors.